### ISSUE TITLE
Flashlight Still not working (Not a Beta anymore)!

### ISSUE THREAD

**Issue by akhillincoln01** [nate-parrott/Flashlight#546]
Please Help me!?
I really want this app to work....Im running OS X 10.10.4

**Comment by CodeBrauer** [comment]
look here #537

**Comment by marrie** [comment]
I still can't get flashlight to work under El Capitan. It actually died about 2 months ago but I never bothered to report the issue. Now I get the message that my spotlight is too new.

**Comment by CodeBrauer** [comment]
Yeah, because Flashlight is currently "dead"

**Comment by geeeeeeeeek** [comment]
Actually the author said in [#537](https://github.com/nate-parrott/Flashlight/issues/537) that "Hacking Spotlight is impossible in 10.11". And it seems that he wants Flashlight to be independent of Spotlight. So I guess it won't be possible to see it working again.

**Comment by iancrowther** [comment]
Please update the readme to explain this app will no longer work in "El Capitan"

**Comment by marrie** [comment]
Or just take it down as it quit working since about may or so. How  sad as I was using it for a lot of stuff including disabling wifi and turning it back on, ejecting drives and so forth. I even told about it in my book but said that in time this app because of spotlight's syntax would no longer be needed eventually.

**Comment by briantully** [comment]
Doesn't work in Yosemite 10.10.5 either

**Comment by marrie** [comment]
Yeah that's when it died for me then to. Do you think it's a good idea for this project to be pulled from github? It's a shame as turning wifi on and off, and bluetooth, dialing a phone number or what not was really handy.
